student.rb

```diff
@@ -1,6 +1,9 @@
 require './person'
+require './classroom'
 
 class Student < Person
+  attr_accessor :name, :classroom
+
   def initialize(name, age, classroom)
     super(name, age)
     @classroom = classroom
```

Comment:
- [ ] Please kindly ensure that there is a method that serves as the 'belongs-to' side (a student belongs to a classroom). Something like this 
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
- [ ] Also, please ensure that the classroom only has a getter exposed since we have a setter method for it.
`attr_reader :classroom`